kong/cmd/start.lua

```diff
@@ -18,7 +18,10 @@ local function execute(args)
   local dao = assert(DAOFactory.new(conf))
   xpcall(function()
     assert(prefix_handler.prepare_prefix(conf, args.nginx_conf))
-    assert(dao:run_migrations())
+    if args.run_migrations then
+      assert(dao:run_migrations())
+    end
+    assert(dao:are_migrations_uptodate())
```

Comment:
The `are_migrations_uptodate` will log a message with `warn` level. Still it seems like here it is a fatal. I would probably move that log statement out of that `are_migrations_uptodate` and do the logging where it has a context.

If we want this to prevent Kong from starting, it should be logged with possibly `> ngx.ERR`?